src/citations/index.ts

```diff
@@ -1,7 +1,30 @@
+import { markdown } from "../lib/utils"
 import { Citation } from "./CitationsTypes"
 
 export type Citations = Citation[]
 
+export function cite(cit: Citation): string {
+  const pa = cit.publisher || cit.authors
+  const prefix = pa != undefined ? pa + ": " : ""
+  const title = cit.title || cit.url.toString()
+  const url = cit.url.toString()
+  return `[[${prefix}${title}](${url})]`
+}
+
+export function citationsDescription(citations: Citations): string {
+  let result: string = ""
+  for (const cit of citations) {
+    result =
+      result +
+      (cit.title ? '"' + cit.title + '"' : "(no title)") +
+      (cit.publisher ? ", " + cit.publisher : "") +
+      ", " +
+      cit.url +
+      "; "
+  }
+  return result
+}
```

Comment:
Wenn wir das übersichtlicher machen wollen, könnten wir die "Aufbereitung" der einzelnen Komponenten in Variablen nach vorne ziehen und dann mittels eines Template-Strings die Komponenten zusammen fügen. Dann wird uns Prettier auch nicht so lästig fallen. Was meinst du?

Reply:
So könnte das aussehen:
```
export function citationsDescription(citations: Citations): string {
  return citations
    .map(citation => {
      const title = citation.title ? '"' + citation.title + '"' : "(no title)"
      return [title, citation.publisher, citation.url].filter(a => a).join(", ")
    })
    .join("; ")
}
```

Reply:
Wenn wir dann noch die Funktion `toString` nennen und mit ins `Citation` Objekt rein stecken, kann man sie einfach im String-Kontext (z.B. im Template) verwenden.

Reply:
Ließe sich aber auch alles nachträglich im separaten PR machen. Also gern auch mergen.

Reply:
Oh ja. Schön. Mach ich - wobei die eh nur für schöne Unit Tests verwendet wird...